data.json

```diff
@@ -142,7 +146,6 @@
       }
     },
     "parser": {
-      "completeness": "intent"
```

Comment:
@surma Do I understand correctly that this field was added mistakenly? `"parser"` should consist of browsers, which in turn include `"completeness"` field. Removing `parser.completeness` doesn't affect resulting HTML